package.json

```diff
@@ -13,7 +13,11 @@
     "lint:fix": "wsrun -c -s lint:fix",
     "build": "wsrun -e -c -s build",
     "test": "wsrun -e -c -s --exclude-missing test",
-    "clean": "wsrun -e -c -s clean"
+    "clean": "wsrun -e -c -s clean && rimraf node_modules"
+  },
+  "resolutions": {
+    "js-waku": "0.30.0",
+    "protons-runtime": "3.1.0"
```

Comment:
⚠️ ensures single occurrence and resolve issue with decoding BigInts causing skipping of messages